tests/acceptance/test_mender-connect.py

```diff
@@ -277,14 +290,18 @@ def test_mender_connect_reconnect(
         """Test that mender-connect can re-establish the connection on remote errors"""
```

Comment:
Much of what I commented in the previous test case is applicable to this one, so I won't review it in detail just yet :eyes: